cli/src/main/java/org/infinispan/cli/impl/AeshRuntimeRunner.java

```diff
@@ -0,0 +1,122 @@
+package org.infinispan.cli.impl;
+
+import java.io.IOException;
+import java.util.Set;
+
+import org.aesh.command.AeshCommandRuntimeBuilder;
+import org.aesh.command.Command;
+import org.aesh.command.CommandException;
+import org.aesh.command.CommandNotFoundException;
+import org.aesh.command.CommandRuntime;
+import org.aesh.command.impl.registry.AeshCommandRegistryBuilder;
+import org.aesh.command.parser.CommandLineParserException;
+import org.aesh.command.registry.CommandRegistryException;
+import org.aesh.command.validator.CommandValidatorException;
+import org.aesh.command.validator.OptionValidatorException;
+import org.aesh.readline.ShellImpl;
+import org.aesh.readline.tty.terminal.TerminalConnection;
+import org.aesh.terminal.Connection;
+
+public class AeshRuntimeRunner {
+
+   private Class<? extends Command> command;
+
+   private CommandRuntime runtime;
+
+   private String[] args;
+   private boolean interactive = false;
+
+   private AeshRuntimeRunner() {
+   }
+
+   public static AeshRuntimeRunner builder() {
+      return new AeshRuntimeRunner();
+   }
+
+   public AeshRuntimeRunner command(Class<? extends Command> command) {
+      this.command = command;
+      return this;
+   }
+
+   public AeshRuntimeRunner commandRuntime(CommandRuntime runtime) {
+      this.runtime = runtime;
+      return this;
+   }
+
+   public AeshRuntimeRunner interactive(boolean interactive) {
```

Comment:
This method is never used and so interactive is always false, is this an oversight?

Reply:
I copied the class 1:1 from Aesh.